src/main.py

```diff
@@ -25,71 +25,89 @@
 
 @app.route("/")
 def index():
-    return render_template("index.html", the_title="Paint Drying")
+	return render_template("index.html", the_title="Paint Drying")
+
+
+@app.route("/login", methods=['POST', 'GET'])
+def login():
+	global current_user_email
+
+	form = LoginForm()
+	if form.validate_on_submit():
+		users = db.get_clients(lambda client: client["email"] == form.email.data)
+		if not users:
+			raise Exception('User with given email does not exist')
+		user = users[0]
+		# user data is ready to further processing
+
+		current_user_email = form.email.data
+
+		return redirect(url_for('index'))
+	return render_template("login.html", form=form, the_title="Login - Paint Drying")
 
 
 @app.route("/register", methods=['POST', 'GET'])
 def register():
-    global current_user_email
```

Comment:
what happened to indentation from this point onward?

Reply:
Now should be ok